src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -171,7 +171,6 @@ const FilterableWorkflowTable = ({
           },
         },
         [
-          div([h2(['Workflows'])]),
```

Comment:
If the `h2` is removed this page, then headings that are currently `h3` should become `h2`s so that no levels are skipped.
https://www.a11yproject.com/posts/how-to-accessible-heading-structure/